components/turret.py

```diff
@@ -30,7 +32,8 @@ class Turret:
     control_loop_wait_time: float
 
     # max rotation either side of zero
-    MAX_ROTATION = math.radians(200)
+    MIN_ROTATION = math.radians(-30)
+    MAX_ROTATION = math.radians(390)
```

Comment:
Isn't 0 towards the intake?

Reply:
Yes. Tested on the real hardware and it works.